tools/downloader/requirements-tensorflow.in

```diff
@@ -1,4 +1,4 @@
-tensorflow>=2.3
+tensorflow~=2.4.1
 
 # workaround for https://github.com/tensorflow/tensorflow/issues/44467
 h5py<3
```

Comment:
You can remove this workaround, since the underlying issue was fixed in TensorFlow 2.4.